ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -1826,3 +1827,65 @@ def test_catalog_list_upgradable_packages(
     assert pkg.version == '2015-09'
     assert pkg.size == '200KB'
     assert isinstance(pkg, ZippedZim)
+
+
+def test_catalog_list_nothandled_packages(
+        tmpdir, settings, testdatadir, mocker):
+    from ideascube.serveradmin.catalog import Catalog
+
+    zippedzim = testdatadir.join('catalog', 'wikipedia.tum-2015-08')
+    path = tmpdir.mkdir('source').join('wikipedia_tum_all_nopic_2015-08.zim')
+    zippedzim.copy(path)
+
+    remote_catalog_file = tmpdir.join('source').join('catalog.yml')
+    with remote_catalog_file.open(mode='w') as f:
+        f.write('all:\n')
```

Comment:
Not fan neither, but they were their before :)

I suggest that we open a new bug to fix this f.write in all tests and keep this specific test coherent with others for this pr.

(I'm not against fixing this, even for the next release. But this is two separated things, so two separated pr/bugs)